app/views/customers/show.html.erb

```diff
@@ -0,0 +1,11 @@
+Name: <%= @customer.name %>
+
+<% @customer.items.each do |item| %>
+<div id=<%="item-#{item.name}"%>>
+  <ul>Item: <%= item.name%></ul>
+  <ul>Price: $<%= item.price%></ul>
+  <ul>From Supermarket: <%= item.supermarket.name%></ul>
+</div>
+<% end %>
+
+Add Item
```

Comment:
You were correct about moving towards putting the form on this page! The `new` page is only required if the user story says that you need to click a button to get to a form to create a new object (i.e. the form lives on a separate page). In this case, the form was meant to just live on the show page). Great intuition!

As for the path for the form - that's a tougher question! What you're doing here by adding an item to a customer, is creating a joins record in the customer_items table. you are creating something, so it's a good idea for that form to be a POST. Rather than creating a new customer, or creating a new item, you're just creating the relationship between 2 existing things. The most RESTful option here, then, would be to create a new controller called the CustomerItemsController and you could have a `create` action where you create the joins record. So, your form could look like:
`form_with url: "/customers/#{@customer.id}/items", method: :post . . . ` and then that route would go to this new controller. Once the joins record is made, that controller action could redirect back to this customer show page